cdap-watchdog/src/main/java/io/cdap/cdap/logging/LoggingConfiguration.java

```diff
@@ -31,7 +31,7 @@ public final class LoggingConfiguration {
   public static final String KAFKA_PRODUCER_BUFFER_MS = "kafka.producer.buffer.ms";
 
   public static final String DEFAULT_LOG_PATTERN =
-    "%d{ISO8601} - %-5p [%t:%c{1}@%L] - category_tags: %mdc{error.tags} %m%n";
+    "%d{ISO8601} - %-5p [%t:%c{1}@%L] - %mdc{error.tags} %m%n";
```

Comment:
why do we need it in the log message at all? this looks like internal details getting exposed to the user.

Reply:
For example, here are some verbose logs I see in my program logs from some class that we don't even control:

```
2022-10-25 17:54:00,030 - INFO  [JobGenerator:o.a.k.c.c.i.SubscriptionState@397] - category_tags:  [Consumer clientId=consumer-22-13-Kafka-Pipeline-Read_v2-user-test-new-2, groupId=22-13-Kafka-Pipeline-Read_v2-user-test-new] Resetting offset for partition user-test-new-5 to offset 36015872.
2022-10-25 17:54:00,030 - INFO  [JobGenerator:o.a.k.c.c.i.SubscriptionState@397] - category_tags:  [Consumer clientId=consumer-22-13-Kafka-Pipeline-Read_v2-user-test-new-2, groupId=22-13-Kafka-Pipeline-Read_v2-user-test-new] Resetting offset for partition user-test-new-2 to offset 35820369.
```

Reply:
Are you suggesting we just keep the tags and not have the prefix "category_tags"? I thought it will be easier to parse the tags out. Otherwise we may have to use the location in the string to extract the tags.

Reply:
I thought the purpose of adding it to MDC was so that it comes back in the MDC section of the log response? Or was the plan to parse it from the message? Parsing is much more brittle than just reading it from the response.

Reply:
Thanks. I didn't realize the mdc was in the response. Removed the tags altogether from the string.